presto-docs/src/main/sphinx/troubleshoot.rst

```diff
@@ -0,0 +1,14 @@
+***************
+Troubleshooting
+***************
+
+.. toctree::
+    :maxdepth: 1
+
+    troubleshoot/deploy
+
+This section of the Presto documentation collects known problems when working 
+with Presto, and their identified solutions or workarounds. 
+
+To contribute to this section, or to any other part of the Presto documentation, use the 
+*Create docs issue* in the right margin of this page.
```

Comment:
So they would just create a doc issue, describe the problem and solution, then someone will review and add it to the repo right?